cfg.sample.toml

```diff
@@ -10,6 +10,23 @@ access_token = ""
 app_client_id = ""
 app_client_secret = ""
 
+# A base shared secret; this can be used to generate the per-repo github secret,
+# which is what's used to verify authenticity of github -> Homu webhooks. The
+# per-repo secret can be computed as follows (you'll need to enter this in
+# the repo webhook configuration):
+#
+# $ echo -n "$owner.$name" | openssl sha1 -hmac "$secret"
```

Comment:
OK, I thought about this and I see the point.  Still though, unlike a main motivation for HKDF, we're *not* trying to stretch weak key material.  There's no reason not to make the base secret as strong as you want - the suggested 160 bits is by any reasonable measure quite strong.

The main thing that's having me hesitate still is it needs to be easy for users to script.  It'd be easier if we shipped with Homu a script that idempotently sets the webhook configuration for a set of repos, given a config file.  Then we wouldn't need to expose the user too much to our secret calculation scheme.